src/main/java/baritone/process/FarmProcess.java

```diff
@@ -169,6 +170,13 @@ private boolean isNetherWart(ItemStack stack) {
 
     @Override
     public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
+        Set<Item> validDrops = new HashSet<Item>(PICKUP_DROPPED);
```

Comment:
<Item> I don't think is needed, just <> should work fine afaik.

Reply:
this works for `HashSet<>` but not for `Set<Item>`